### ISSUE TITLE
[subplat] Privacy Policy link in HTML transactional emails is missing

### ISSUE THREAD

**Issue by data-sync-user** [mozilla/fxa#8031]
While I was working on FXA-3114, I noticed that, while we include the product's terms of service link in both HTML and plain text emails, we don't include a link to the product's privacy policy in HTML emails. These links are to the localized PDF versions of the documents.

[~byee@mozilla.com] I wanted to surface this for your thoughts on how high a priority this bug is (e.g. from a legal perspective).

I should note that we do link to the ToS and Privacy Policy webpages during Checkout on the Subscription Platform website, though we only provide a downloadable PDF link for the ToS and not the Privacy Policy for some reason.

┆Issue is synchronized with this [Jira Bug](https://jira.mozilla.com/browse/FXA-3290)
┆Attachments: <a href="https://jira.mozilla.com/secure/attachment/15713/Checkout_footer.png">Checkout_footer.png</a> | <a href="https://jira.mozilla.com/secure/attachment/15712/HTML_email_footer.png">HTML_email_footer.png</a> | <a href="https://jira.mozilla.com/secure/attachment/15916/privacy_tos.JPG">privacy_tos.JPG</a>
┆Issue Number: FXA-3290

**Comment by data-sync-user** [comment]
➤ Barry Chen commented:

While working on this I noticed that we do have a privacy notice link!  It's at the bottom to the right of the "Legal" link.

**Comment by data-sync-user** [comment]
➤ Barry Chen commented:

The privacy notice link is in the HTML emails already.

**Comment by data-sync-user** [comment]
➤ Bianca Danforth commented:

Oh interesting; I would imagine that link should actually go to Mozilla's privacy statement, as opposed to a product-specific privacy policy? The Legal link next to it goes to https://www.mozilla.org/about/legal/terms/services/, so wouldn't we want that link to go to e.g. https://www.mozilla.org/en-US/privacy/?

**Comment by data-sync-user** [comment]
➤ Barry Chen commented:

That'd make sense to me.  But let's see what Product/Legal think?

[~byee@mozilla.com]  The "Privacy" link in the attached HTML_email_footer.png (lower right, below the Mozilla logo) is a link to the product specific privacy policy doc.  Should that be a link to Mozilla's privacy policy page and the link for the product specific doc would go after "Terms and cancellation policy" (above the Mozilla logo)?

**Comment by data-sync-user** [comment]
➤ Brian Yee commented:

So there is a:

  1. VPN privacy notice:  https://www.mozilla.org/en-US/privacy/mozilla-vpn/ ( https://www.mozilla.org/en-US/privacy/mozilla-vpn/ )
  2. Mozilla privacy policy:  https://www.mozilla.org/en-US/privacy/? ( https://www.mozilla.org/en-US/privacy/? )
  3. VPN terms of service:   https://www.mozilla.org/en-US/about/legal/terms/mozilla-vpn/ ( https://www.mozilla.org/en-US/about/legal/terms/mozilla-vpn/ )
  4. Mozilla terms of service:  https://www.mozilla.org/about/legal/terms/services/ ( https://www.mozilla.org/about/legal/terms/services/ )

From what I gather, your questions are:

Should the footer bottom right link to privacy go to #1 or #2?

Should the "terms and cancellation policy" go to #3 or #4?

[~bdanforth@mozilla.com] [~bchen@mozilla.com] Please confirm those are the questions and I will follow up with legal.  My hunch would be for product-specific for terms (since that's related to the product),  company specific for privacy (since that's the footer)

**Comment by data-sync-user** [comment]
➤ Bianca Danforth commented:

Thanks Brian.

For my part, I was asking: How critical is it (i.e. MVP or not) to have the product specific privacy policy (#1) linked in our e-mails? This is because I thought it wasn't there, while all the other legal doc links were (#2, #3 and #4).

Barry pointed out that #1 is in our emails where you would expect #2 to be. As a consequence, it turns out #2 is actually missing from our emails. So Barry is asking the first question in your list: Should the footer bottom right link to privacy go to #1 or #2?

Overall, I think our observations lead us to ask: How critical (i.e. MVP or not) is it for us to include links to some or all of these legal documents (#1, #2, #3 and #4) in a) emails and b) on the website? Currently, we are not very consistent.

**Comment by data-sync-user** [comment]
➤ Brian Yee commented:

I pinged Feldman on this so will let you know when I hear.

**Comment by data-sync-user** [comment]
➤ Brian Yee commented:

Hi team,

See this link for a summary of what should be linked to where:

https://docs.google.com/presentation/d/1EcVbHjUj-vdJDUE2tZotnr7t3VnZcSv4GbQ2UAN1uS4/edit?usp=sharing ( https://docs.google.com/presentation/d/1EcVbHjUj-vdJDUE2tZotnr7t3VnZcSv4GbQ2UAN1uS4/edit?usp=sharing )

Note that checkout flow footer links have to be right (blocker), but email links are non-blocking, medium-high priority, fast-follow fixes.

**Comment by data-sync-user** [comment]
➤ Bianca Danforth commented:

Thanks Brian for following up with this. Currently, what we have in the emails for the "Privacy" link is the product-specific privacy policy, #1, not #5. So that sounds like a fast-follow to me. We'll leave this ticket open to fix this.

**Comment by data-sync-user** [comment]
➤ Bianca Danforth commented:

Note: we should use the URL for the website privacy notice that directs users to a localized version of the webpage rather than https://www.mozilla.org/en-US/privacy/websites/ . Trying this out myself, it looks to be: https://www.mozilla.org/privacy/websites/ .

**Comment by data-sync-user** [comment]
➤ Barry Chen commented:

Below is an example of what the current plaintext (we always send both html and plaintext) email looks like (sorry Jira formatting sucks).  Note that we do not have the link to the website ToS or Privacy Notice; instead it has links to the product specific ToS and Privacy notice PDFs.  [~byee@mozilla.com]  can you confirm that is okay?

Cooking with Foxkeh payment confirmed

Your payment is currently processing and may take up to four business days to complete.

You will receive a separate email with download instructions on how to start using Cooking with Foxkeh.

Your subscription will automatically renew each billing period unless you choose to cancel.

Invoice Number: CD3195E7-0001

Charged $2.00 on 04/26/2021

Visa card ending in 4242

Next Invoice: 05/03/2021

Questions about your subscription? Our support team is here to help you:

http://localhost:3030/support?utm_medium=email&utm_campaign=fx-subscription-first-invoice&utm_content=fx-subscription-support

This is an automated email; if you received it in error, no action is required.

Terms and cancellation policy:
http://localhost:3031/legal-docs?url=https%3A%2F%2Faccounts-static.cdn.mozilla.net%2Flegal%2FMozilla_VPN_ToS%2Fen-US.pdf%3Futm_medium%3Demail%26utm_campaign%3Dfx-subscription-first-invoice%26utm_content%3Dfx-subscription-terms

Privacy notice:
http://localhost:3031/legal-docs?url=https%3A%2F%2Faccounts-static.cdn.mozilla.net%2Flegal%2Fmozilla_vpn_privacy_notice%2Fen-US.pdf%3Futm_medium%3Demail%26utm_campaign%3Dfx-subscription-first-invoice%26utm_content%3Dfx-subscription-privacy

Cancel subscription:
http://localhost:3030/subscriptions?plan_id=price_1ITxqUBVqmGyQTMafJzCcz2y&product_id=prod_GvH2k78kKusAlV&uid=75c37094501d4609ba89a6947690d7fd&email=test%40testo.com&utm_medium=email&utm_campaign=fx-subscription-first-invoice&utm_content=fx-cancel-subscription

Update billing information:
http://localhost:3030/subscriptions?plan_id=price_1ITxqUBVqmGyQTMafJzCcz2y&product_id=prod_GvH2k78kKusAlV&uid=75c37094501d4609ba89a6947690d7fd&email=test%40testo.com&utm_medium=email&utm_campaign=fx-subscription-first-invoice&utm_content=fx-update-billing

Mozilla Corporation
2 Harrison St, #175, San Francisco, CA 94105

**Comment by data-sync-user** [comment]
➤ Valentina Virlics commented:

Hello everyone, after reading the above comments, I am a bit confused about what links should be displayed in both subplat and emails. 
 Below you can see what we currently show.
 1. In the “Mozilla VPN payment confirmed” email we currently have the redirections:

  * “Terms and cancellation policy” - downloads Mozilla VPN tos (localized);
  * “Privacy” - https://www.mozilla.org/en-US/privacy/websites/ ( https://www.mozilla.org/en-US/privacy/websites/ );
!Screenshot 2021-05-06 11.55.46.png|thumbnail!

2. In the Subscription Platform pages (Set up your subscriptions, Thank you pages) we currently have the following redirections:
 a) At authorization checkbox

  * “Terms of Service” - https://www.mozilla.org/en-US/about/legal/terms/mozilla-vpn/ ( https://www.mozilla.org/en-US/about/legal/terms/mozilla-vpn/ )
  * “Privacy Notice” - https://www.mozilla.org/en-US/privacy/mozilla-vpn/ ( https://www.mozilla.org/en-US/privacy/mozilla-vpn/ )

b) Under “Pay now” button

  * “Terms of Service” - https://www.mozilla.org/en-US/about/legal/terms/mozilla-vpn/ ( https://www.mozilla.org/en-US/about/legal/terms/mozilla-vpn/ )
  * “Download Terms” - downloads Mozilla VPN tos (localized)
  * “Privacy Notice” - https://www.mozilla.org/en-US/privacy/mozilla-vpn/ ( https://www.mozilla.org/en-US/privacy/mozilla-vpn/ )
!Screenshot 2021-05-06 11.56.35.png|thumbnail!

As a side note, I encountered a strange behavior, on my Linux machine, the “Download Terms” are downloaded in French even if the browser and the system language are in en-US. It is important to mention that the language of the system/browser was set in the past in French, but not anymore. The “Download Terms” for FPN for example, are in English, so I am inclined to think that it is an issue somewhere.
 !tosfr.png|thumbnail!

**Comment by data-sync-user** [comment]
➤ Bianca Danforth commented:

??As a side note, I encountered a strange behavior, on my Linux machine, the “Download Terms” are downloaded in French even if the browser and the system language are in en-US. It is important to mention that the language of the system/browser was set in the past in French, but not anymore. The “Download Terms” for FPN for example, are in English, so I am inclined to think that it is an issue somewhere.??

That's odd. [~vvirlics@mozilla.com] , to clarify, your browser's language setting in `about:preferences` is set to use English? Do you see this on Windows or Mac as well?

I haven't been able to reproduce on my Mac, but please feel free to file a separate issue here with more detailed steps, and we can investigate.

**Comment by data-sync-user** [comment]
➤ Valentina Virlics commented:

So, I manage to reproduce it only on Linux, but I am almost sure that I am missing something that I did on Lnix, and did not on Windows.

I do not have clear steps to reproduce this, that is why I added just a comment here, and not a separate issue.

The Language in about:preferences was set to English, but the French language was there too, o the third position. After I removed French everywhere, the Terms got back to English. So, it is related to some previous settings, or language order, or I don't know.

Regading the Privacy links, we are ok with those that are curently showed (see my comment)? Thanks!
